src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/ComplexCompositeType.cs

```diff
@@ -0,0 +1,275 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace WcfService
+{
+    [DataContract(Namespace = "http://www.contoso.com/wcfnamespace")]
+    public class ComplexCompositeType : IEquatable<ComplexCompositeType>
```

Comment:
I think we've already discussed this and said code duplication was to be avoided.  I agree we don't want to clean up old mistakes in this PR, but this PR already is a fundamental redesign of the location of these kinds of files. Don't block this PR on this discussion, but I recommend when the client PR becomes available we don't duplicate code.